articles/cognitive-services/Speech-Service/speech-container-faq.yml

```diff
@@ -163,14 +163,9 @@ sections:
       - question: |
           Is CentOS supported for Speech containers?
         answer: |
-          CentOS 7 is not supported by Python SDK yet, also Ubuntu 19.04 is not supported.
+          Speech containers should run on most Operating Systems, but the speech SDK is required for the container to operate.
```

Comment:
Casing: "Speech containers should run on most operating systems, but the Speech SDK is required for the container to operate."

Also it seems like this part is not specifically about CentOS and not critical to answering the question: "but the Speech SDK is required for the container to operate" .. They should find such details here: https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/speech-container-howto?tabs=stt%2Ccsharp%2Csimple-format

Suggested: "Speech containers run on most operating systems, including some CentOS configurations. For more information about supported environments, see [Get the Speech SDK](speech-sdk?tabs=linux%2Cubuntu%2Cios-xcode%2Cmac-xcode%2Candroid-studio#get-the-speech-sdk)."

Reply:
I applied the requested edit.